content/v2/csidriver/upgradation/drivers/powermax.md

```diff
@@ -24,10 +24,11 @@ You can upgrade CSI Driver for Dell PowerMax using Helm or Dell CSI Operator.
 3. Run the `csi-install` script with the option _\-\-upgrade_ by running: 
    ```bash
 
-   cd ../dell-csi-helm-installer && ./csi-install.sh --namespace powermax --values ./my-powermax-settings.yaml --upgrade
+   cd ../dell-csi-helm-installer && ./csi-install.sh --namespace powermax --values ./my-powermax-settings.yaml --upgrade --upgrade --helm-charts-version
```

Comment:
**--upgrade** flag is repeated twice and there is no placeholder for the helm version like [this ](https://github.com/dell/csm-docs/pull/1200/files#diff-136e2295f8d4f604568b856dfbc781b5cf80f2ef226278377662226d678d073aR22)